src/EventStore.Client/EventStoreClientConnectivitySettings.cs

```diff
@@ -20,12 +20,14 @@ public class EventStoreClientConnectivitySettings {
 		public DnsEndPoint[]? DnsGossipSeeds { get; set; }
 		public IPEndPoint[]? IpGossipSeeds { get; set; }
 		public TimeSpan GossipTimeout { get; set; }
+		public bool UseHttps { get; set; } = true;
```

Comment:
Perhaps we should rename `UseHttps` to make it more obvious that it only applies to gossip seeds, as it doesn't affect single node connection defaults right now.

As it stands, it looks like I could use the following to connect to a single node running with defaults and `--insecure`:

```
ConnectivitySettings =
{
    UseHttps = false
}
```

But this results in SSL errors and I actually have to set the uri :

```
ConnectivitySettings =
{
    Address = new Uri("http://localhost:2113")
}
```

Reply:
The other thing we can do here is infer that from `Address`.

Reply:
@hayley-jean @thefringeninja good catch thanks , i'll try to see if we can infer it from Address so that it can be used both for single nodes and clusters

Reply:
@hayley-jean @thefringeninja what do you think about:
```
		private bool _useHttps = true;
		public bool UseHttps {
			get{
				return _useHttps;
			}
			set{
				_useHttps = value;
				Address = new UriBuilder{
					Scheme = _useHttps ? Uri.UriSchemeHttps : Uri.UriSchemeHttp,
					Host = Address.Host,
					Port = Address.Port
				}.Uri;
			}
		}
```

Reply:
@hayley-jean @thefringeninja after discussing with @pgermishuys , `GossipOverHttps` might be a better name - technically speaking, it is not completely valid since we also use the flag to connect the the node (not only for gossip info) but it seems more intuitive from a user perspective and is consistent with `GossipSeeds` and `GossipTimeout` options

Reply:
Fixed in this commit: 983861de6cbca9d4ee29ea29e008b3469574d5d6